### PR TITLE
feature/BNGP-526-upload-geospatial-land-boundary

### PR DESCRIPTION
**Pull request by pwadmore-ea**
https://eaflood.atlassian.net/browse/BNGP-526

Refactor initial geospatial land boundary upload support and incorporate design changes.

### CODE REVIEW

File: packages/webapp/src/routes/land/upload-geospatial-file.js [hunk 1]
```diff
@@ -70,6 +69,107 @@ const buildSignalRConfig = (sessionId, config) => {
...
+const performUpload = async (request, h) => {
+  const config = buildConfig(request.yar.id)
+
+  try {
+    const geospatialData = (await uploadFiles(logger, request, config))
+    logger.log(`${new Date().toUTCString()} Received land boundary data for ${geospatialData[0].location.substring(geospatialData[0].location.lastIndexOf('/') + 1)}`)
+    request.yar.set(constants.redisKeys.GEOSPATIAL_LOCATION, geospatialData[0].location)
+    request.yar.set(constants.redisKeys.LAND_BOUNDARY_MAP_CONFIG, geospatialData[0].mapConfig)
+    request.yar.set(constants.redisKeys.GEOSPATIAL_FILE_NAME, geospatialData.filename)
+    request.yar.set(constants.redisKeys.GEOSPATIAL_FILE_SIZE, parseFloat(geospatialData.fileSize).toFixed(4))
```
Comment (teddmason) [inline packages/webapp/src/routes/land/upload-geospatial-file.js:81]: Does this need setting to 2dp to be consistent with other file upload pages?

File: packages/webapp/src/routes/land/upload-geospatial-file.js [hunk 2]
```diff
@@ -70,6 +69,107 @@ const buildSignalRConfig = (sessionId, config) => {
   }
 }
 
+const performUpload = async (request, h) => {
+  const config = buildConfig(request.yar.id)
+
+  try {
+    const geospatialData = (await uploadFiles(logger, request, config))
```
Comment (teddmason) [inline packages/webapp/src/routes/land/upload-geospatial-file.js:76]: Why is the await uploadFiles() wrapped in parentheses? I think they can be removed, unless there is a programmatic reason I don't know of?

Comment (pwadmore-ea) [reply packages/webapp/src/routes/land/upload-geospatial-file.js]: Refactored

**Review by teddmason — DISMISSED** [review]

**Review by teddmason — APPROVED** [review]

File: packages/webapp/README.md [hunk 3]
```diff
@@ -23,7 +23,10 @@ warning: other service dependencies will be missing, install and run from root o
 | SESSION_COOKIE_PASSWORD | Password for the session cookie | N |
 | UPLOAD_PROCESSING_TIMEOUT_MILLIS | Upload processing timeout in milliseconds (defaults to 30000) | N |
 | AZURE_FUNCTION_APP_URL | API URL for funtion app | Y |
+<<<<<<< HEAD
```
Comment (teddmason) [inline packages/webapp/README.md:26]: Got some git merge stuff in the readme.md

**Review by teddmason — CHANGES_REQUESTED** [review]
Just the git merge message on the readme file needs sorting

**Review by teddmason — APPROVED** [review]

**Pull request merged**